src/core/Toast/Toast.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { baseStyles } from './Toast.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+import { HtmlDiv, HtmlDivWithRef } from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+
+export interface ToastProps {
+  /** Set aria-live mode for the Toast text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Label heading for the Toast */
+  headingText?: string;
+  /** Main content of the Toast */
+  children?: ReactNode;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Elements to be rendered under the Toast text. Can be e.g. buttons, links etc. */
```

Comment:
This description seems to be off. Children is not explicitly documented, but I assume it should be to give more information on allowed child types. Also, composition could have probably worked here as well, leaving all Heading props out of the main component API. Especially because headingText is not mandatory.

Reply:
Fix. now JS Doc is following for children:
/** Text content of the Toast. */